ionoscloud/data_source_vpn_ipsec_gateway.go

```diff
@@ -79,6 +79,28 @@ func dataSourceVpnIPSecGateway() *schema.Resource {
 				Description: "The IKE version that is permitted for the VPN tunnels.",
 				Computed:    true,
 			},
+			"maintenance_window": {
```

Comment:
Is it possible to have multiple maintance_window objects?

Reply:
@digna-ionos no, it is not possible. For resources we use `MaxItems: 1` to restrict the creation, but for data sources there is no need to define `MaxItems: 1` since in the read function only one `maintenance_window` will be read.